docs/src/content/docs/intro/why.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Why Filecoin Onchain Cloud?
+description: Learn about the key benefits and use cases of Filecoin Onchain Cloud for decentralized storage and services.
+sidebar: 
+  order: 2
+---
+
+Traditional cloud storage solutions present several challenges:
+
+- **Centralized Control**: Data and services are controlled by a single provider
+- **Limited Transparency**: Operations and pricing are opaque
+- **Vendor Lock-in**: Difficult to migrate between providers
+- **Trust Requirements**: Users must trust providers to handle data properly
+- **Fixed Payment Models**: Inflexible billing tied to specific providers
+
+## The FOC Solution
+
+Traditional cloud services offer performance and usability but lack transparency, auditability, and user ownership. **Filecoin Onchain Cloud** bridges this gap by introducing a **trustless execution layer** for decentralized services. Each service in the Onchain Cloud — from storage to payment — operates as a smart contract, ensuring that every action, transaction, and proof is **verifiable on the Filecoin blockchain**.
+
+FOC is designed to:
+- ✅ **Enable hot and verifiable data storage and retrieval** with Proof of Data Possession (PDP) & Filecoin Beam
+- 💰 **Power programmable payments** via Filecoin Pay — a token-agnostic payment rail for decentralized services
+- 🧩 **Simplify developer integration** through the Synapse SDK — a TypeScript library for building Filecoin-native dApps
+- 💻 **Provide programmable services** — Smart contract-based, composable service primitives on top of the Filecoin network
+- 🚀 **Foster innovation** by letting teams build, fork, and combine services into new decentralized applications
+
+### Key Differentiators
+
+| Feature | Traditional Cloud | Filecoin Onchain Cloud |
+|---------|------------------|------------------------|
+| Data Control | Provider controlled | User owned |
+| Transparency | Limited | Full blockchain verification |
+| Programmability | Provider APIs | Smart contracts |
+| Payment Models | Fixed | Flexible & programmable |
```

Comment:
Not a strong concern for me, but I could see a developer wondering whether this means what *they pay and how they pay* for the primary offering, which is the warm storage, is flexible to them.  I think developers want clarity on how they pay for the service.

I believe this is intended for in the event the developer creates their own service and want to bill their own customers, they can do that with Filecoin Pay?

If so, can we frame it as we enable them to bill their customers flexibly?  Versus they can pay for the Storage Service we provide out of the box flexibly?

Reply:
This payment models is just general point of paying the Filecion warm storage service. At this point, we take developers as our audience regardless how they bill their own customers. 

In terms of developers want clarity on how they pay for the service, this will be covered in the developer guides under payment rails.